resources/resources.md

```diff
@@ -0,0 +1,16 @@
+---
+nav_order: 121
+has_children: true
+title: Resources
+description: Machine translation resources
+---
+
+### Calls for papers
```

Comment:
Maybe make this its own article?

Then the auto-generated TOC here will list it.